src/test/java/dev/jbang/TestScript.java

```diff
@@ -209,6 +209,36 @@ public static void writeContentToFile(Path path, String content) throws IOExcept
 		}
 	}
 
+	private void addUrlToTrustedSources(String url) {
```

Comment:
this just looks wrong - trustedsources shouldn't be public and mutated from outside.

Reply:
yes. It feels wrong, but I'm changing just the test
the variable was already public =D
At least it allows a very good test

Reply:
there is both an add and remove method with storage as option so just need  to try/finally that - no need to expose internal logic. I've pushed change that avoids this.

Reply:
Great!